.github/workflows/pr-clean-deployments.yml

```diff
@@ -8,6 +8,9 @@ on:
     types: [closed]
     branches: [main, release/**, next/**]
 
+permissions:
```

Comment:
Given that we write to the gh-pages branch in here I think this needs `contents: write` too